locales/en-US/app-en-US.json

```diff
@@ -73,6 +73,14 @@
     "err" : "Error exporting \"__file__\", please report this issue with the problem project file.",
     "note" : "Successfully exported \"__file__\" for printing."
   },
+  "tracing": {
+    "menutitle": "Import image for tracing…",
```

Comment:
Should probably specify as we do above, "Import image for _automatic_ tracing..."